src/diag/perf_mon.cpp

```diff
@@ -87,7 +82,7 @@ esp_err_t init_perfmon(void)
 
 void remove_perfmon(void)
 {
-    timer_deinit(TIMER_GROUP_0, TIMER_1);
+    //gptimer_disable(gptimer);
```

Comment:
I would expect that line not to be a comment.

Reply:
Your right! Resolved in d3ad19d22fae508a6f375b3a12e583e511b7925a